ContributingToMC.md

```diff
@@ -0,0 +1,172 @@
+
+# Submitting a patch to Firefox using Git.
+
+This guide will take you through submitting and updating a patch to `mozilla-central` as a git user. You need to already be [set up to use git to contribute to `mozilla-central`](MozCentralQuickStart.md).
+
+## Performing a bug fix
+
+All of the open bugs for issues in Firefox can be found in [Bugzilla](https://bugzilla.mozilla.org). If you know the component that you wish to contribute to you can use Bugzilla to search for issues in that project. If you are unsure which component you are interested in, you can search the [Good First Bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=good-first-bug) list to find something you want to work on. 
+
+* Once you have your bug, assign it to yourself in Bugzilla.
+* Update your local copy of the firefox codebase to match the current version on the servers to ensure you are working with the most up to date code.
+
+```bash
+git remote update
+```
+* Create a new feature branch tracking either Central or Inbound.
+
+```
+git checkout -b bugxxxxxxx [inbound|central]/default
+```
+* Work on your bug, checking into git according to your preferred workflow. _Try to ensure that each individual commit compiles and passes all of the tests for your component. This will make it easier to land if you use `moz-phab` to submit (details later in this post)._
+
+You must have Mozilla commit access in order to submit your fix directly to the Firefox repository. There are three levels of commit access that give increasing levels of access to the repositories.
+
+Level 1: Try/User access. You will need this level of access commit to the try server. 
+Level 2: General access. This will give you full commit access to any mercurial or SVN repository not requiring level 3 access.
+Level 3: Core access. You will need this level to commit to any of the core repositories (Firefox/Thunderbird/Fennec).
+
+If you wish to apply for commit access, please follow the guide found in the [Mozilla Commit Access Policy](https://www.mozilla.org/en-US/about/governance/policies/commit/access-policy/).
+
+If you do not have access and still want to submit your fix, you can create a patch and attach it directly to the Bugzilla ticket.
+
+### Attaching a patch in Bugzilla
```

Comment:
We should encourage people to use Phabricator for all patches. https://moz-conduit.readthedocs.io/en/latest/phabricator-user.html